cytosnake/helpers/helper_funcs.py

```diff
@@ -0,0 +1,71 @@
+"""
+Module: helper_funcs.py
+
+This module contains helper functions for snakemake workflows to access with. 
```

Comment:
```suggestion
This module contains helper functions for snakemake workflows. 
```

Please edit my suggestion if it removes your meaning